src/test/java/org/reactome/release/common/dataretrieval/TestFileRetrieval.java

```diff
@@ -91,18 +91,18 @@ public void testFetchData() throws Exception
 			
 			String dest = "/tmp/testFetchData_"+ String.valueOf((new Random()).nextInt(Integer.MAX_VALUE));
 			retriever.setFetchDestination(dest);
-			Duration age = Duration.of(5,ChronoUnit.SECONDS);
+			Duration age = Duration.of(5, ChronoUnit.SECONDS);
 			retriever.setMaxAge(age);
 			
 			retriever.fetchData();
 			assertTrue(Files.exists(Paths.get(dest)));
 			
-			//Sleep for two seconds, and then re-download because the file is stale
-			Thread.sleep(Duration.of(2,ChronoUnit.SECONDS).toMillis());
+			//Sleep for two seconds, and then re-download because the file is stale.
```

Comment:
"Two" should be changed to "Six" to be in sync with the code

Reply:
:facepalm: